test/unit/features/component/component-async.spec.js

```diff
@@ -253,22 +253,22 @@ describe('Component async', () => {
       }
     })
 
-    it('with error component', done => {
+    it('with error component + a prop named error', done => {
       const vm = new Vue({
         template: `<div><test/></div>`,
         components: {
           test: () => ({
             component: new Promise((resolve, reject) => {
               setTimeout(() => {
-                reject()
+                reject('error')
```

Comment:
Errors are usually `new Error('message')`, so better test it that way so it is closer to real scenarios